pkg/securitypolicy/regopolicy_test.go

```diff
@@ -3925,6 +3925,35 @@ func Test_Rego_MissingEnvList(t *testing.T) {
 	}
 }
 
+func Test_Rego_EnvListGetsRedacted(t *testing.T) {
+	c := generateConstraints(testRand, 1)
+	// don't allow env dropping. with dropping we aren't testing the
+	// "invalid env list" message, only the input
+	c.allowEnvironmentVariableDropping = false
+
+	// No environment variables are allowed
+	//c.containers[0].EnvRules = make([]EnvRuleConfig, 0)
+	tc, err := setupRegoCreateContainerTest(c, c.containers[0], false)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	var envList []string
+	envVar := "FOO=BAR"
+	envList = append(envList, envVar)
+
+	_, _, err = tc.policy.EnforceCreateContainerPolicy(tc.sandboxID, tc.containerID, tc.argList, envList, "bunk", tc.mounts, false)
+
+	// not getting an error means something is broken
+	if err == nil {
+		t.Fatal("Unexpected success when enforcing policy")
+	}
+
+	if strings.Contains(err.Error(), envVar) {
+		t.Fatal("EnvList wasn't redacted in error message")
+	}
```

Comment:
we should also check the error contains `FOO=<<redacted>>`, right?

Reply:
i think that is an overspecialization, but Maksim would also like to see, so I am adding.